spec/features/card_spec.rb

```diff
@@ -33,4 +37,11 @@
       end
     end
   end
+
+  context 'When there are not card for review' do
+    it 'shows message "All cards reviewed"' do
+      visit root_path
+      expect(page).to have_content('Все карточки пересмотрены')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.